java/gazelle/testdata/lib_with_complex_runtime_deps/README.md

```diff
@@ -0,0 +1,5 @@
+# Complex runtime_deps
+
+Loading existing runtime_deps attribute only supports string list values.
+
+Other type are ignored and deleted.
```

Comment:
```suggestion
Other type are ignored and deleted.

Note that this is not the desired behaviour, simply documenting the current behaviour. Improving this would be welcome.
```